staging/src/k8s.io/component-base/metrics/testutil/metrics.go

```diff
@@ -35,7 +35,9 @@ var (
 	// MetricNameLabel is label under which model.Sample stores metric name
 	MetricNameLabel model.LabelName = model.MetricNameLabel
 	// QuantileLabel is label under which model.Sample stores latency quantile value
-	QuantileLabel model.LabelName = model.QuantileLabel
+	QuantileLabel                             model.LabelName = model.QuantileLabel
```

Comment:
This is probably causing the golint error (causing the hack/verify failure)

Reply:
actually, it's not, the golint error from another place, thanks anyway for the review.